rust/src/quic/quic.rs

```diff
@@ -103,6 +109,10 @@ pub struct QuicState {
     state_data: AppLayerStateData,
     max_tx_id: u64,
     keys: Option<QuicKeys>,
+    /// crypto fragment data already seen and reassembled to client
+    frag_tc: Vec<u8>,
```

Comment:
would suggest naming this `crypto_frag_*`

Reply:
ok